scicat/access_individual_dataset.ipynb

```diff
@@ -0,0 +1,1820 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9ac76e6f-c896-43b7-bdd6-5808b22bc7b8",
+   "metadata": {},
+   "source": [
+    "# Access Individual Dataset\n",
+    "## DMSC Summer School\n",
+    "  \n",
+    "This notebook contains an example on how to load a single dataset from SciCat, view it and retrieve the data files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49c39c47-c770-499a-a0d0-205fab11d274",
+   "metadata": {},
+   "source": [
+    "Load standard libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8a773922-7b44-428d-a79c-d59d39bb4282",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0694c31-d5ef-4111-bc3d-b639b0da2a02",
+   "metadata": {},
+   "source": [
+    "URL of the scicat instance containing the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9cf209fe-04aa-4a11-b432-a8a3f2eca7e5",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "scicat_instance = \"https://staging.scicat.ess.eu/api/v3\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1fcebbba-a5eb-4678-a160-fbce79c2e34c",
+   "metadata": {},
+   "source": [
+    "Valid Authentication token  \n",
+    "_To obtain the token, log in on your scicat instance and go to User->settings page._"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "aecee7a2-9017-402f-addb-4779734ad313",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "token=\"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2MzliMmE1MWI0MTU0OWY1M2RmOWVjMzYiLCJyZWFsbSI6ImxvY2FsaG9zdCIsInVzZXJuYW1lIjoiaW5nZXN0b3IiLCJlbWFpbCI6InNjaWNhdGluZ2VzdG9yQHlvdXIuc2l0ZSIsImVtYWlsVmVyaWZpZWQiOnRydWUsImF1dGhTdHJhdGVneSI6ImxvY2FsIiwiaWQiOiI2MzliMmE1MWI0MTU0OWY1M2RmOWVjMzYiLCJpYXQiOjE2OTIwODc0ODUsImV4cCI6MTY5MjA5MTA4NX0.Phca4UF7WKY367-10Whgwd5jaFjiPku6WsgiPeDh_-o\""
```

Comment:
Please remove!

Reply:
This is an expired token.
Would it be acceptable to leave in it as an example?

Reply:
It suggests bad practice. I'd still replace it with something like `"your-token-here"`.

Reply:
The way you changed it should be fine